domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.GetMeteringPoints.verified.txt

```diff
@@ -0,0 +1,20 @@
+﻿{
+  Result: [
+    {
+      GSRN: 1234567890123456,
```

Comment:
I think we need to tell if it is production or consumption

Reply:
It comes in the response but it is not mapped out to this verified file. The same goes SubMeterType. Do you know why?

Reply:
Made separate assert for this